src/main/java/ladder/domain/LadderGame.java

```diff
@@ -0,0 +1,24 @@
+package ladder.domain;
+
+import java.util.Collections;
+import java.util.List;
+
+public class LadderGame {
+    private static final String ALL = "all";
+
+    private final Participants participants;
+    private final Ladder ladder;
+
+    public LadderGame(Participants participants, Ladder ladder) {
+        this.participants = participants;
+        this.ladder = ladder;
+    }
+
+    public List<Integer> play(String name) {
+        if(name.toLowerCase().equals(ALL)) {
```

Comment:
다음 방법으로도 변경해볼수 있을것 같아요!
```suggestion
        if (ALL.equalsIgnoreCase(name)) {
```

Reply:
equalsIgnoreCase 라는 함수가 있었군요.^^